### DISCUSSION TITLE
Lesson 6 Error (6275): Source "src/SimpleStorage.sol"  not found

### DISCUSSION THREAD

**Discussion by Joel-eth** [Cyfrin/foundry-full-course-cu#1643] [General]
I encountered this error while trying to compile the deploy script.

![● SimpleStorage  sol - hello_foundry  WSL_ Ubuntu  - Visual Studio Code 4_19_2024 7_07_44 PM](https://github.com/Cyfrin/foundry-full-course-f23/assets/113260217/2a5bde06-7d3b-4458-ae49-91c033490dbd)

**Comment by Joel-eth** [comment]
please I need help on how to resolve it.

**Reply by EngrPips** [reply]
Hello @Joel-eth, You are trying to import a file that doesn't exist. Check your `src` folder and see if there is any file named `Counter. sol` in it.

**Reply by Joel-eth** [reply]
There isn't
How can I get the file?

**Reply by EngrPips** [reply]
You don't need the `Counter.sol` file for anything what you need to do is open your test folder and delete the `Counter.t.sol` file

**Reply by Joel-eth** [reply]
Thank you bro I just realised that I was calling a non existent file.

**Reply by EngrPips** [reply]
Awesome, I am glad you figured that out. Keep crushing it, friend.
